apps/src/music/SoundsPanel.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './soundsPanel.module.scss';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+
+const getSoundRowClassName = (currentValue, folderPath, soundSrc) => {
+  if (currentValue === folderPath + '/' + soundSrc) {
+    return styles.soundRowSelected;
+  } else {
+    return styles.soundRow;
+  }
+};
+
+const getPreviewClassName = (playingPreview, folderPath, soundSrc) => {
```

Comment:
minor - could these just be ternaries inline if they're only used in one place? maybe we can pull out `folderPath + '/' + soundSrc` into a local to streamline it